shared/js/content-scope/gpc-protection.js

```diff
@@ -3,19 +3,21 @@ import { defineProperty } from './utils'
 // Set Global Privacy Control property on DOM
 export function init (args) {
     try {
-        // If GPC on, set DOM property to true if not already true
+        // If GPC on, set DOM property prototype to true if not already true
         if (args.globalPrivacyControlValue) {
             if (navigator.globalPrivacyControl) return
-            defineProperty(navigator, 'globalPrivacyControl', {
-                value: true,
+            defineProperty(Navigator.prototype, 'globalPrivacyControl', {
+                get: () => true,
+                configurable: false,
```

Comment:
The intention of the existing code seems to align with `configurable: false`, so I've submitted the PR with this value initially. I suggest using `configurable: true` instead so that other extensions may override this value with different implementations.

Reply:
@dharb Thanks, I've gone ahead and updated the `configurable` value 👍